.gitignore

```diff
@@ -68,3 +68,11 @@ util/Win_Check_Output.txt
 secrets.tar
 id_rsa_*
 /.vs
+*.BAK001
```

Comment:
You should probably use `.git/info/exclude` for these instead

Reply:
Could you revert the changes to this file? 

If you need to ignore these files, then the above file that I mentioned will accomplish that, as well.

Reply:
Easy way to revert it is with `git checkout upstream/master .gitignore`, but only if you have an upstream set.